elastic/kibana
# [ML] Increase trial max ram for ml nodes

## Summary
Currently in cloud, trials include only up to 1GB of ram for ML nodes but with the newest NLP models the nodes will require to have between 800MB and 1.3GB of ram. In order to encourage users to try the new models we're increasing the maximum allowed ram for free trial users to 2GB. See: https://github.com/elastic/cloud/issues/100018 for more details.

**How to test**
1. Start up elasticsearch, `yarn es snapshot --license=trial`
2. Configure your `config/kibana.yml` to have cloud `base_url` and `deployment_url`
3. Make sure you dont have any ML nodes running locally
3. Start up Kibana, `yarn start` and navigate to ML overview page
4. Verify that the `NodeAvailableWarning` callout shows and and it mentions enabling a free 2GB ML node
